gkeys/gkeys/email_reminders.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import print_function
```

Comment:
the file name email_reminders is too specific.  The Emailer class should be generic in that it is passed the template and the data to use, it just puts it together and sends/queues it.  So email.py is sufficient.